src/sentry/static/sentry/app/views/settings/incidentRules/details.tsx

```diff
@@ -143,15 +144,14 @@ class IncidentRulesDetails extends AsyncView<
   };
 
   renderBody() {
-    const {orgId, incidentRuleId} = this.props.params;
+    const {orgId, projectId, incidentRuleId} = this.props.params;
     const {rule} = this.state;
 
     return (
-      <div>
-        <SettingsPageHeader title={t('Edit Incident Rule')} />
-
+      <React.Fragment>
```

Comment:
I'm guessing that this is needed because `<RuleForm>` creates a list of children?

Reply:
It's needed because it wraps 3 elements (`RuleForm`, `TriggersHeader`, and `TriggersList`)